srcpkgs/brother-hl5340d-cupswrapper/REMOVE

```diff
@@ -0,0 +1,6 @@
+case ${ACTION} in
+    pre) 
+	usr/lib/brother/cupswrapper/cupswrapperHL5340D-2.0.4 -e
+    	rm -f usr/share/cups/model/BR5340_2_GPL.ppd
```

Comment:
I wouldn't recommend deleting files that were just placed by the package manager.